Regression/WarpX-tests.ini

```diff
@@ -1558,6 +1558,23 @@ compareParticles = 0
 analysisRoutine = Examples/Tests/reduced_diags/analysis_reduced_diags_loadbalancecosts.py
 tolerance = 1e-12
 
+[reduced_diags_loadbalancecosts_heuristic_omp]
+buildDir = .
+inputFile = Examples/Tests/reduced_diags/inputs_loadbalancecosts
+runtime_params = warpx.do_dynamic_scheduling=0 warpx.serialize_ics=1 algo.load_balance_costs_update=Heuristic warpx.load_balance_efficiency_ratio_threshold = 1e-5 max_step = 10
+dim = 3
+addToCompileString =
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 1
+numthreads = 4
```

Comment:
Careful in CI: We only have two vCores available - so using 8 will likely trigger a watchdog deamon.
Does this also crash locally?

Side note: working with OLCF & NERSC to get some beefier CI machines for such tests integrated...